emojiHeatmapper/lib/js/util.js

```diff
@@ -0,0 +1,225 @@
+'use strict';
+
```

Comment:
I have used already existing libraries. You can check it [here](https://github.com/OneSignal/emoji-picker). @Achint08